_includes/chapter2.html

```diff
@@ -827,7 +843,7 @@ <h3>Creating Docker worker images that don't require additional privileges</h3>
     To deploy the worker, configure a <b>Deploy Kubernetes containers</b> step with the YAML shown below, replacing the <b>ServerUrl</b> and <b>Space</b> environment variables with your own Octopus server and space. Note that this container does not have the <b>privileged</b> option set, and disables Docker-in-Docker by setting the <b>DISABLE_DIND</b> environment variable to <b>Y</b>:
 </p>
 <p class="note">
-    We have used the system variable <b>Octopus.Space.Name</b> in the example below assuming the workers are being deployed to the same space the runbook is run from.
+    We have used the system variable <b>Octopus.Space.Name</b> in the example below, assuming the workers are being deployed to the same space the runbook is run from.
```

Comment:
We have used the system variable <b>Octopus.Space.Name</b> in the example below, assuming the workers are being deployed to the same space as the runbook.